package/MDAnalysis/core/AtomGroup.py

```diff
@@ -1143,6 +1234,7 @@ def n_segments(self):
         return len(self.segments)
 
     @property
+    @warn_atom_property
```

Comment:
``*Group.atoms.indices``

Indices always refer to atom indices (zero-based, and always unique among atoms), no matter what level called from. In order to get back indices as they currently work from all levels (except AtomGroup) in the new system, one must prefix with ``.atoms``.